src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/ProjectExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.CodeAnalysis;
+
+static class ProjectExtensions
+{
+    public static Project AddDocuments(this Project project, IEnumerable<string> files)
+    {
+        foreach (string file in files)
+        {
+            project = project.AddDocument(file, File.ReadAllText(file)).Project;
+        }
+        return project;
+    }
+
+    private static IEnumerable<string> GetAllSourceFiles(string directoryPath)
+    {
+        var filePaths = Directory.GetFiles(directoryPath, "*.cs", SearchOption.AllDirectories);
+        return filePaths;
+    }
```

Comment:
put all helper prviate method below public ones that use them - easier to navigate

Reply:
fixed.